packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -122,59 +145,407 @@
 </section>
 
 <section>
-  <h3 class="dummy-h3">Temporary scratch showcase</h3>
+  <h3 class="dummy-h3">How to use</h3>
+  <h4 class="dummy-h4">
+    Basic invocation: Alert (Page)
+  </h4>
+  {{! prettier-ignore-start }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Alert @title="Title here" @description="Description here" />
+    '
+  />
+  {{! prettier-ignore-end }}
+
+  <p class="dummy-paragraph">Renders to the default
+    <code class="dummy-code">type</code>
+    of
+    <code class="dummy-code">page</code>
+    and
+    <code class="dummy-code">color</code>
+    of
+    <code class="dummy-code">neutral</code>:
+  </p>
+  <Hds::Alert @title="Really cool title here" @description="And a rad description too!" />
+  <br />
+  <a href="#showcase-page">Jump to Showcase: Alert (Page)</a>
```

Comment:
Let's not add this new pattern here. If you think is necessry, let's open an issue to add it to every component page.

Reply:
0350cb1dce06bc08923453f180715d4b35f34775